redundant-bmc/src/rbmc_manager_main.cpp

```diff
@@ -0,0 +1,29 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+#include "manager.hpp"
+
+#include <sdbusplus/async/context.hpp>
+#include <sdbusplus/server/manager.hpp>
+#include <xyz/openbmc_project/State/BMC/Redundancy/common.hpp>
+
+using Redundancy =
+    sdbusplus::common::xyz::openbmc_project::state::bmc::Redundancy;
+
+int main()
+{
+    sdbusplus::async::context ctx;
+    sdbusplus::server::manager_t objMgr{ctx, "/xyz/openbmc_project/state"};
+
+    rbmc::Manager manager{ctx};
+
+    // clang-tidy currently mangles this into something unreadable
+    // NOLINTNEXTLINE
+    ctx.spawn([](sdbusplus::async::context& ctx) -> sdbusplus::async::task<> {
+        ctx.request_name("xyz.openbmc_project.State.BMC.Redundancy");
```

Comment:
I had that defined initially, but Patrick had a comment here https://gerrit.openbmc.org/c/openbmc/phosphor-dbus-interfaces/+/70765/1..5/yaml/xyz/openbmc_project/State/BMC/Redundancy.interface.yaml#b42 and told me to remove it, though I didn't really understand the reasoning. in fact it even seems to conflict his documentation.

Reply:
Yeah, it's a bit confusing to me as well. I might check with him again during the data sync regarding the YAML file. 

However, I noticed that it matches the [interface name](https://github.com/ibm-openbmc/phosphor-dbus-interfaces/blob/1110/yaml/xyz/openbmc_project/State/BMC/Redundancy.interface.yaml). Should we use something like Redundancy::interface?

(I know it’s a bit confusing, so either approach works since we likely won’t need to change this much.)

Reply:
I guess the interface constant is used in other places in the repo:
```
$ git grep request_name | grep interface
bmc_state_manager_main.cpp:    bus.request_name(BMCState::interface);
chassis_state_manager_main.cpp:        bus.request_name(ChassisState::interface);
host_state_manager_main.cpp:        bus.request_name(HostState::interface);
scheduled_host_transition_main.cpp:        bus.request_name(ScheduledHostTransition::interface);
scheduled_host_transition_main.cpp:    bus.request_name((std::string{ScheduledHostTransition::interface} +
```
so I may as well do that here too, even though it is kinda confusing.